src/e2e/__tests__/caret.ts

```diff
@@ -39,7 +36,14 @@ describe('caret testing', () => {
 
     await page.keyboard.press('Enter')
     await paste(page, [''], importText)
-    await setCursor(page, ['puppeteer', 'web scrapping'], { offset: 3 })
+
+    await clickBullet(page, 'puppeteer')
```

Comment:
Thanks, this is an improvement over setting the `cursor` directly.

Clicking the bullet usually has the same effect as clicking on the editable, but it's more of an edge case, so I'd prefer not to use it for our primary means of moving the cursor around. (In many outliners you click the bullet to expand/collapse an item, but that's not the case in **em**. In **em**, you move the cursor and the bullets auto-expand/collapse for you.)

You could create a `clickThought` helper or just use `(await getEditable(page, value)).click()`.